vulcanai/models/basenetwork.py

```diff
@@ -646,7 +656,7 @@ def fit(self, train_loader, val_loader, epochs,
                 train_loss, train_acc = self._train_epoch(train_loader,
                                                           retain_graph)
 
-                valid_loss = valid_acc = np.nan
+                valid_loss = valid_acc = None
```

Comment:
nice - it's good to have this updated since we had such trouble deciding what to leave it as. However, I think there may be issues when it comes to plotting the curves with matplotlib. I would double check that the val loss still lines up with the epochs that it's actually calculated at

Reply:
yep it destroys the tests. now fixed by converting back to np.nan after the early stopping tests are done... refactor suggestions appreciated, although it's fine as is

Reply:
It's something specific with how matplotlib prints it so when we migrate to a better visualization library we might be able to handle this better so it's not immediately alarming because nan is not a great place holder for score values in the world of DL :P

Reply:
haha yep agreed